updates/v1/entrypoint_release.json

```diff
@@ -28,5 +28,10 @@
         "version": "6.0.3",
         "severity": "Critical",
         "time": "2023-04-15T12:00:00Z"
+    },
+    {
+        "version": "6.1.0",
+        "severity": "Major",
+        "time": "2023-04-21T12:00:00Z"
```

Comment:
Is that a bit in the future? April 21.

Reply:
I did it for purpose as review time is always at least one day, then we wait for distribution and then merging release notes